src/toast/tests/accelerator.py

```diff
@@ -134,36 +137,41 @@ def test_memory(self):
 
         # Copy to device
         for tname, buffer in data.items():
-            accel_data_create(buffer)  # TODO buffer = acel... in the jax case?
-            accel_data_update_device(buffer)
+            buffer = accel_data_create(buffer)
+            buffer = accel_data_update_device(buffer)
+            data[tname] = buffer
 
         # Check that it is present
         for tname, buffer in data.items():
             self.assertTrue(accel_data_present(buffer))
 
         # Change host copy
         for tname, buffer in data.items():
+            # FIXME we are changing the host copy while data is on device
+            #       is it legal?!
```

Comment:
These lines are just testing the low-level data movement functions.  There are checks in the `AcceleratorObject` methods against this kind of thing.